src/test/java/org/sagebionetworks/bridge/models/DateTimeHolderTest.java

```diff
@@ -0,0 +1,25 @@
+package org.sagebionetworks.bridge.models;
+
+import static org.sagebionetworks.bridge.TestConstants.TIMESTAMP;
+import static org.testng.Assert.assertEquals;
+
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class DateTimeHolderTest {
+    
+    @Test
+    public void canSerialize() throws Exception {
+        DateTimeHolder holder = new DateTimeHolder(TIMESTAMP);
```

Comment:
We should use a timestamp with a non-UTC timezone to make sure our serialization also handles timezones correctly.